groupy/update.ps1

```diff
@@ -13,20 +13,16 @@ function global:au_SearchReplace {
 
 function global:au_GetLatest {
     $releases = Invoke-WebRequest -Uri $history_page
-
-    $tag = "h2"
-    $re = "\w[Groupy]\s\d"
-    $ver = "\d\.\d{0,}(\n|$|\.\d{0,})"
-
-    $version = (($releases.AllElements | Where-Object { $_.TagName -eq $tag -and $_.innerText -Match $re }).innerText | Select-Object -First 1) -Match $ver
+    
     $url = "https://cdn.stardock.us/downloads/public/software/groupy/Groupy_setup.exe"
-    # Old $url including the specific version. Keeping for now as Stardock changes their URL scheme often.
-    # $url = ( -Join ("https://cdn.stardock.us/downloads/public/software/groupy/Groupy", $Matches[0], "_setup_sd.exe") )
+
+    $re = "Groupy (?<version>[\d\.]+[\d\.]+)"
+    $version = $releases -match $re | ForEach-Object { $Matches.version }
 
     return @{
         URL32   = $url
-        Version = $Matches[0]
+        Version = $version
     }
 }
 
-Update-Package
+Update-Package -ChecksumFor none
```

Comment:
Why would we not update the checksum? At least with `groupy`, it did change with every release in the past since the installation package is not downloaded by the installer.

Reply:
The `au_SearchReplace` functions framework automagically generates a checksum for the file and injects it into the `$global:Latest` variable before writing it to the `chocolateyInstall.ps1`.  When it gets to the `Update-Package` command, sometimes invoking `chocolateyInstall.ps1` during the automatic checksum can be problematic so it needs to be disabled using the `ChecksumFor none` parameter, thus increasing the speed at which the script executes.